### PR TITLE
Small text changes and code updates

### PR DESCRIPTION
**Pull request by kostadin-kanbanize**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`.

### CODE REVIEW

**Comment by v-EgorKozhadei** [comment]
Hi @kostadin-kanbanize!

Thank you very much for updating the branch!
Right now connector files are in root directory. Please move connector files to the appropriate folder `certified-connectors/Kanbanize` and remove icon.png and settings.json. These files are not required for the review.
The folder should contain 4 files: apiProperties.json, apiDefinition.swagger.json, readme.md, script.csx (optional).

**Comment by kostadin-kanbanize** [comment]
Sorry for this. I have deleted the files from the root and moved them to the correct folder. 

Thank you for your continuous and quick assistance.

File: certified-connectors/Kanbanize/apiDefinition.swagger.json [hunk 1]
```diff
@@ -2368,25 +2301,25 @@
...
                   "type": "string",
-                  "description": "taskid",
+                  "description": "card id",
                   "title": "Card ID",
                   "x-ms-visibility": "important"
                 },
                 "history": {
                   "type": "string",
-                  "description": "Set to “yes” if you want to get history for the task",
+                  "description": "set to \u201cyes\u201d if you want to get card history",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Kanbanize/apiDefinition.swagger.json:2310]: @kostadin-kanbanize unicode symbols are not allowed. Please remove them. You can use apostrophe sign instead, e.g. `"Set to 'yes' if..."`

File: certified-connectors/Kanbanize/apiDefinition.swagger.json [hunk 2]
```diff
@@ -2454,8 +2379,8 @@
               "properties": {
                 "boardid": {
                   "type": "string",
-                  "description": "boardid",
-                  "title": "Board",
+                  "description": "-- select board --",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Kanbanize/apiDefinition.swagger.json:2382]: @kostadin-kanbanize please make sure the description looks like a sentence. If there's a dropdown list, you can ask the user to select the value, e.g. `"Select a board from the dropdown list"`

**Review by v-EgorKozhadei — COMMENTED** [review]
Hi @kostadin-kanbanize!

Please check comments after the review. There're a few minor things left.

File: certified-connectors/Kanbanize/apiDefinition.swagger.json (hunk 1 above)
Comment (kostadin-kanbanize) [reply certified-connectors/Kanbanize/apiDefinition.swagger.json]: Unicode characters removed

File: certified-connectors/Kanbanize/apiDefinition.swagger.json (hunk 2 above)
Comment (kostadin-kanbanize) [reply certified-connectors/Kanbanize/apiDefinition.swagger.json]: Description updated

**Comment by kostadin-kanbanize** [comment]
I have applied the needed updates.

**Review by v-EgorKozhadei — APPROVED** [review]

**Comment by v-EgorKozhadei** [comment]
Hello @kostadin-kanbanize, 

I hope you are doing well. 

Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) using your latest commit id, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file. 

After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6. 

Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.  

We expect all tests to be completed within 1-2 weeks. 

If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions. 

Please let us know if you have any questions. 

Thank you very much for working with us.

**Pull request merged**